dist/rpm/openQA.spec

```
@@ -215,6 +217,13 @@ Requires(pre):  group(kvm)
 %description worker
 The openQA worker manages test engine (provided by os-autoinst package).
 
+%package mcp
+Summary:        Additional MCP package for AI support in openQA
+Requires:       %{mcp_requires}
+
+%description mcp
+This package contains additional resources for AI support in openQA.
```

If we merge the actual plugin we can rewrite this:
```suggestion
This package contains a plugin for AI support in openQA.
```

---

Good point. I included that in https://github.com/os-autoinst/openQA/pull/6711